### DISCUSSION TITLE
Switch to GitHub Actions

### DISCUSSION THREAD

**Discussion by Spectre5** [robbievanleeuwen/section-properties#60] [Ideas]
@robbievanleeuwen what are you thoughts on switching from travis CI to the native github actions?  After we add some lint/code quality checks to the repository, I'll follow up with another PR to add those checks to the CI pipeline.  Travis is fine, I'm just more familiar with github actions.  I'm mostly curious if there is a specific reason you're using travis instead?

**Comment by robbievanleeuwen** [comment]
Hi @Spectre5,

No specific reason that I used TravisCI, was just something that I'd heard of and figured out how to work.

I like the idea of moving to github to keep everything 'in-house'! It will also be nice to have some familiarity with it when we implement the code quality checks!

I'll make an issue out of this and we can address it with v2!

Robbie
